src/order/filter/pointwise.lean

```diff
@@ -1,3 +1,16 @@
+/-
+Copyright (c) 2019 Zhouhang Zhou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Zhouhang Zhou
+
+Pointwise addition and multiplication of filters.
+
+Has nice properties, such as
```

Comment:
Maybe turn this into a standalone sentence. Something along the lines of "The pointwise operations on filters have nice properties, such as".